pkg/server/api/entry/v1/service_test.go

```diff
@@ -37,6 +37,68 @@ var (
 	agentID           = spiffeid.RequireFromString("spiffe://example.org/agent")
 )
 
+func TestCountEntries(t *testing.T) {
+	for _, tt := range []struct {
+		name    string
+		count   int32
+		code    codes.Code
+		err     string
+		logMsg  string
+		dsError error
+	}{
+		{
+			name:  "0 entries",
+			count: 0,
+		},
+		{
+			name:  "1 entries",
+			count: 1,
+		},
+		{
+			name:  "2 entries",
+			count: 2,
+		},
+		{
+			name:  "3 entries",
+			count: 3,
+		},
+		{
+			name:    "ds error",
+			err:     "failed to count entries: ds error",
+			code:    codes.Internal,
+			logMsg:  "Failed to count entries",
+			dsError: errors.New("ds error"),
+		},
+	} {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			ds := fakedatastore.New(t)
+			test := setupServiceTest(t, ds)
+			defer test.Cleanup()
+
+			for i := 0; i < int(tt.count); i++ {
+				entry, err := test.ds.CreateRegistrationEntry(ctx, &datastore.CreateRegistrationEntryRequest{
```

Comment:
`entry` is not used you can just do `_, err := test.ds.....`

Reply:
It's currently checked for not nil; however the check for no error is probably sufficient